social/communication_and_social_stuff.md

```diff
@@ -0,0 +1,48 @@
+# Communication and Social Stuff
+## Email Lists/ Google Groups
+- saturn-dev
+- dsp-analysis-platform
+
+## Calendars
+- DSDE Events
+
+## Slack Channels
+Some of these are more work-related than others. Here are some recommendations, but definitely explore the entire menu on Slack!
+
+- 105b
+- a11y
+	- general channel for discussing accessibility
+- broadexchange
+- crafters
```

Comment:
I think this list of slack channels might be a bit robust, perhaps just listing the most general fun ones and then mentioning something like "there is the option in slack to search all channels and that there are lots more to see" would make sense?

Reply:
Actually I would suggest checking out how @Shakespeared formatted the slack channels in her PR that got merged, I think the categories add a lot of value. 

With this PR there is now duplicate info so I would recommend either linking here to the slack.md file and removing this slack content or moving the contents of that file to replace this slack section here. Thoughts? @blrubenstein @Shakespeared

Reply:
TOL: I think whether the slack info should be its own file depends on how much other similar info we have. My feeling is that we don't want any one document to get too overwhelmingly long or have too much disparate information in it, because we want it to be easy to find what you need in the docs. 

As it is now I think it could be in this doc, but if we want to build out more of the social/culture stuff then it might make sense to keep it separated. I also think the categories are helpful - especially for the team/work related channels, and it's nice to have a brief description as well (otherwise the channel names can be quite mysterious).

Reply:
Fair enough!